x/distribution/client/testutil/cli_test.go

```diff
@@ -12,7 +13,7 @@ import (
 
 func TestIntegrationTestSuite(t *testing.T) {
 	cfg := network.DefaultConfig()
-	cfg.NumValidators = 1
+	cfg.NumValidators = 2
```

Comment:
why do we need to run 2 vals?

Reply:
We need delegations to unique validators to test withdraw all rewards tx with `--maxmsgs` flag.